docs/rules/use-selector-in-select.md

```diff
@@ -1,6 +1,6 @@
 # Use Selector In Select
 
-> Using a selector in a select method is preferred in favor of strings or props drilling.
+> Using a selector in a select method is preferred over string or props drilling.
```

Comment:
I was copying (😅) this description to #248 and realized that "over" fits better than "in favor of" in this case, so I decided to change this too. Let me know if it's fine for you too.